### PR TITLE
Implemented Broadcast component

### PR DESCRIPTION
**Pull request by benja**
The broadcast component features:

- Fetching the broadcast endpoint with the current window location as path
- Automatically refetches broadcast endpoint and updates or removes broadcast based on response
- Customisable props: `interval`, `path`, `dev` to adjust behavior

### CODE REVIEW

File: pages/components/broadcast.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+          <f-broadcast dev="true"></f-broadcast
+        ></syntax-highlight>
+        <syntax-highlight>
+          // 4 second refetch interval
+          <f-broadcast interval="4000"></f-broadcast>
+        </syntax-highlight>
+        <syntax-highlight>
+          // The path is automatically set through window.location.href, but you
+          can override this by passing path as a param
+          <f-broadcast path="finn.no/finn/travel"></f-broadcast>
```
Comment (digitalsadhu) [inline pages/components/broadcast.html:38]: Should this be just /finn/travel? Path/pathname usually implies the stuff after the origin.

File: pages/components/broadcast.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        </syntax-highlight>
+
+        <f-broadcast
+          dev="true"
+          path="http:/local.finn.no:8080/trygg-handel/bil?finnkode=2"
```
Comment (digitalsadhu) [inline pages/components/broadcast.html:43]: Missing a slash in the origin here.

File: pages/components/broadcast.html (hunk 1 above)
Comment (benja) [reply pages/components/broadcast.html:38]: I was under the impression, until until I realized [https://dev.finn.no/broadcasts?path=http:/local.finn.no:8080/trygg-handel/bil?finnkode=2](https://dev.finn.no/broadcasts?path=http:/local.finn.no:8080/trygg-handel/bil?finnkode=2) works.

Comment (benja) [reply pages/components/broadcast.html:38]: https://dev.finn.no/broadcasts suggests `?path=finn.no/finn/travel` too.

Comment (runarb) [reply pages/components/broadcast.html:38]: Rename `path` to `url`, perhaps?

Comment (benja) [reply pages/components/broadcast.html:38]: Renaming to url seems to be more clear, so can do.

Comment (runarb) [reply pages/components/broadcast.html]: Even though the underlying APi says `path` it seems to be some kind of expression: 
image
So, what you pass in depends on how the page(s) is defined, really. Might be a path, url, expression. 🤷🏻

**Pull request merged**